client/me/purchases/credit-cards/payment-method-delete-dialog/style.scss

```diff
@@ -0,0 +1,17 @@
+.payment-method-delete-dialog {
+	.button {
+		margin-right: 10px;
+		margin-bottom: 8px;
+		padding: 8px 10px;
+		text-align: left;
+	}
+
+	// &.dialog {
```

Comment:
Is there a reason this is commented-out? Maybe I'll find out in a later commit but I'm mentioning it here anyway.